mmdet/datasets/pipelines/transforms.py

```diff
@@ -2802,6 +2802,7 @@ def __init__(
         self.bbox_occluded_thr = bbox_occluded_thr
         self.mask_occluded_thr = mask_occluded_thr
         self.selected = selected
+        self.mask_gen = False
```

Comment:
Add mask_gen in parameters，and claim that copypaste base on bboxes is equal to cutmix for detection in docstrings.

Reply:
Add mask_gen in __repr__